doc/api/cli.md

```diff
@@ -423,6 +423,15 @@ interoperability with non-conformant HTTP implementations. It may also allow
 request smuggling and other HTTP attacks that rely on invalid headers being
 accepted. Avoid using this option.
 
+### `--jitless`
+<!-- YAML
+added: v12.0.0
+-->
+
+Disable [runtime allocation of executable memory][jitless]. This may be
+required on some platforms for security reasons. It can also reduce attack
+surface on other platforms, but the performance impact may be severe.
```

Comment:
While there are exceptions we generally, as a rule, do not document v8 specific options as those are not generally covered by any stability guarantees. @devsnek any thoughts on this?

Reply:
I think it's safer to assume this flag will remain stable than other flags we have on the list, like the `--perf-*` family (which are explicitly not supported by V8).

On the other hand, the criteria to add a flag to that list is not clear or documented (afaik). We should probably document it. Or maybe we shouldn't have V8 flags in our API documentation, and we should point users to `--v8-options` instead?

Reply:
I think it's okay to document --jitless, but we shouldn't make a habit of it.

Reply:
I think we should make it clear in the docs that this flag comes from V8 and is subject to change in the upstream.